src/pretix/base/models/event.py

```diff
@@ -81,8 +81,11 @@ class Event(LoggedModel):
     live = models.BooleanField(default=False, verbose_name=_("Shop is live"))
     permitted = models.ManyToManyField(User, through='EventPermission',
                                        related_name="events", )
+    CURRENCY_CHOICES = [(settings.CURRENCIES[i].alpha_3, settings.CURRENCIES[i].alpha_3 + " - " + settings.CURRENCIES[i].name)
```

Comment:
Two comments here. First, can you move that further above whithin this class? I don't think it's usual to mix constants and fields. Best put it before all fields and seperated with an empty line from the fields.

Second, iterating over ``range(0, len(…))`` is not really idiomatic python when we can have
```python
CURRENCY_CHOICES = [(c.alpha_3, c.alpha_3 + " - " + c.name) for c in settings.CURRENCIES]
```
instead.